ros/ackermann_vehicle_navigation/scripts/angle_converter.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import rospy
+from geometry_msgs.msg import Twist
+from std_msgs.msg import UInt16
+from ackermann_msgs.msg import AckermannDrive
+
+def convert_to_angle(steering_angle, speed):
+    motor_angle=int(99+80*speed)
```

Comment:
These would be more readable if we defined a `constrain(n)` and an `interpolate(x, a, b, c, d)` function (or imported them from somewhere).